utils/config/config_test.go

```diff
@@ -161,6 +166,32 @@ func TestGetArtifactoriesFromConfig(t *testing.T) {
 	}
 }
 
+func TestGetJfrogDependenciesPath(t *testing.T) {
+	// Check default value of dependencies path, should be JFROG_CLI_HOME/dependencies
+	dependenciesPath, err := GetJfrogDependenciesPath()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	jfrogHomeDir, err := GetJfrogHomeDir()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	expectedDependenciesPath := filepath.Join(jfrogHomeDir, JfrogDependencies)
+	if strings.Compare(expectedDependenciesPath, dependenciesPath) != 0 {
+		t.Error(errors.New(fmt.Sprintf("Dependencies Path should be %s (actual path: %s)", expectedDependenciesPath, dependenciesPath)))
+	}
+	// Check dependencies path when JFROG_DEPENDENCIES_DIR is set, should be JFROG_DEPENDENCIES_DIR/
+	err = os.Setenv(cliutils.JFrogCliDependenciesDir, "/tmp/my-dependencies")
+	if err != nil {
+		t.Error(err.Error())
+	}
+	expectedDependenciesPath = "/tmp/my-dependencies/"
```

Comment:
The */tmp/my-dependencies* is defined twice in this function. The variable *expectedDependenciesPath* can be used.